Project - Sara Herranz

sarisham opened the pull request:
Dear Iñaki,
I just uploaded the newest version of my project. I would like to ask you for some advice since I have not been able to solve this issues on my own:

1. In the first replication ('plot'), I don't know why "Other South America" is not represented, despite having more than enough data for inclusion. I would like to write an explanation as to why this happens, but I have not been able to come up with another explanation rather than "geom_stream represents the relative proportions but is less effective for studying the evolution of each individual group". 

2. In the section "Dealing with missing data", I don't know why the graph ('plot2') does not appear when the chunk is executed. It initially worked the first time I tried, but it has been consistently showing up blank thereafter.

4. Since the original graph is in Spanish, I decided to keep the replica in Spanish as wll. Should I change it so it is in English? The alternative visualizations are in English.

5. I restyled my code to fit the width of the column as you suggested, but I was not able to do so when dealing with titles, subtitles and comments. Trying to adjust them seemed to affect the representation of the text in the graph. Could you advise if there is a solution for this, or should I leave it as it is, even tough it does not perfectly math the column width?

Best, 
Sara

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
```diff
@@ -0,0 +1,454 @@
+---
+title: "El crecimiento de la industria petrolera"
```
English, please.

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
```diff
@@ -0,0 +1,454 @@
+    toc: true
+---
+
+git add
+git commit 
+git push
+en algun lugar tengo que añadir _project. 
+es lo que estuvimos haciendo el otro día
+
+
+Distill is a publication format for scientific and technical writing, native to the web.
+
+Learn more about using Distill at <https://rstudio.github.io/distill>.
```
Remember to remove this ;-)

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
````diff
@@ -0,0 +1,454 @@
+library(plotrix)
+```
+
+## **CLEANING THE DATA**
````
Use sentence case for titles and no further styles (i.e. remove bold markers).

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
````diff
@@ -0,0 +1,454 @@
+First, we start by organizing the countries. We created a vector for the countries that needed to appear explicitly in our graph and then created a new database where all these countries appear. After that, we filtered the original database, but this time we selected the unwanted countries and summed them by region. That way, they would receive the name of the region.
+
+```{r}
+countries <- c("Australia", "China", "Indonesia", "Norway", "Russian Federation", "USSR", "United Kingdom", "Brazil", "Venezuela", "Mexico", "Canada", "US", "Saudi Arabia", "Iran", "Kuwait", "Iraq", "Libya", "United Arab Emirates", "Nigeria", "Other Africa", "Other Asia Pacific", "Other CIS", "Other Caribbean", "Other Eastern Africa", "Other Europe", "Other Middle Africa", "Other Middle East", "O ...
````
This line (and others) go out of the page. Please restyle your code to fit in the width of the column (see the resulting HTML. A maximum of 80 characters is generally advisable.

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
````diff
@@ -0,0 +1,454 @@
+
+
+```{r}
+# Modificamos las variables regionales.
+
+  # primero creamos vectores
````
Ensure that all comments, variable names... are in English

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
````diff
@@ -0,0 +1,454 @@
+
+```{r include=FALSE}
+  # defining oilprod_kbd as character in both data bases
+as.character(panel_a$oilprod_kbd)
+as.character(panel_b$oilprod_kbd)
````
Please, remove this. There's a lot of output here that is not useful for the reader.

Enchufa2 commented on _projects/2023/100517886/100517886.Rmd:
```diff
@@ -0,0 +1,454 @@
+panel_c <- bind_rows(panel_a, panel_b)
+
+  # defining oilprod_kbd as numeric
+as.numeric(panel_c$oilprod_kbd)
```
Remove too

Enchufa2 reviewed (changes requested):
Nice work! I gave you some indications of what you got so far. I'm converting this to draft again, because there's some work to be done. Mark it as ready for review again when you push the new version.

sarisham commented on _projects/2023/100517886/100517886.Rmd:
Dear Iñaki,
I just uploaded the newest version of my project. I would like to ask you for some advice since I have not been able to solve this issues on my own:

In the first replication ('plot'), I don't know why "Other South America" is not represented, despite having more than enough data for inclusion. I would like to write an explanation as to why this happens, but I have not been able to come up with another explanation rather than "geom_stream represents the relative proportions but is less effective for studying the evolution of each individual group".

In the section "Dealing with missing data", I don't know why the graph ('plot2') does not appear when the chunk is executed. It initially worked the first time I tried, but it has been consistently showing up blank thereafter.

Since the original graph is in Spanish, I decided to keep the replica in Spanish as wll. Should I change it so it is in English? The alternative visualizations are in English.

I restyled my code to fit the width of the column as you suggested, but I was not able to do so when dealing with titles, subtitles and comments. Trying to adjust them seemed to affect the representation of the text in the graph. Could you advise if there is a solution for this, or should I leave it as it is, even tough it does not perfectly math the column width?

Best,
Sara

Enchufa2 reviewed (changes requested):
> In the first replication ('plot'), I don't know why "Other South America" is not represented, despite having more than enough data for inclusion. I would like to write an explanation as to why this happens, but I have not been able to come up with another explanation rather than "geom_stream represents the relative proportions but is less effective for studying the evolution of each individual group".

I don't know either. What happens when you use `geom_area`? Maybe it's overrepresented in the original graph?

> In the section "Dealing with missing data", I don't know why the graph ('plot2') does not appear when the chunk is executed. It initially worked the first time I tried, but it has been consistently showing up blank thereafter.

Maybe it just doesn't work with the suggestion I made. It seems it doesn't like the 0. With a small value, like 0.001, it works, but the effect is not quite like the original.

> Since the original graph is in Spanish, I decided to keep the replica in Spanish as wll. Should I change it so it is in English? The alternative visualizations are in English.

It's fine. I was just referring to variable names, function names, and so on.

> I restyled my code to fit the width of the column as you suggested, but I was not able to do so when dealing with titles, subtitles and comments. Trying to adjust them seemed to affect the representation of the text in the graph. Could you advise if there is a solution for this, or should I leave it as it is, even tough it does not perfectly math the column width?

It's fine in this way.

Please let me know when this is ready for review again.

sarisham commented:
Dear Iñaki

First, I would like to apologize for the delay on my commit. On Saturday, I was reviewing the final version, and I came up with a way to handle the lack of data in my database. I managed to find a solution, but I spent Sunday trying to improve it without success. Finally, today I focused on tweaking some aesthetic aspects of this new version.

The most significant changes occurred in the section titled "**Addressing some issues. 1. Dealing with missing data**", where I describe the steps taken, and I believe this would be the final replication of the chart. It's the best I could do given the limitations you are already aware of, but I am quite satisfied with the result as it is much better than the previous attempt. I also made a couple of adjustments in "**Alternative visualizations**", but the rest of the work remains the same as in the previous versions.

The only thing that concerns me more is adjusting the dimensions of the charts like "{r, fig.height=8, fig.width=7} and similar ones." I'm unsure if they display correctly, or if I should make any changes.

Thank you very much!

Best, 
Sara

Enchufa2 reviewed (approved):
Awesome, thanks! Merging with minor fixes.

Enchufa2 commented:
Here's your project, congrats! https://csslab.uc3m.es/dataviz/projects/2023/100517886/
